packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1692,13 +1740,14 @@ export class ContainerRuntime
 			disableIsolatedChannels: metadata?.disableIsolatedChannels,
 			gcVersion: metadata?.gcFeature,
 			options: JSON.stringify(runtimeOptions),
+			idCompressorEnabledMetadata: metadata?.idCompressorEnabled,
+			idCompressorEnabled: this.idCompressorMode,
```

Comment:
nit: `idCompressorMode: this.idCompressorMode`. `idCompressorEnabled` sounds like it should be true or false but the value here is a string.